gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -36,6 +35,11 @@
         </dependencies>
```

Comment:
You seem to have 2 versions of `org.springframework.boot`. If you put all of the versions in properties then this can be averted.

Reply:
Done